samples/logging/notifications/sample.md

```diff
@@ -35,9 +35,9 @@ snippet: customSLR
 
 ## Plugging to the API
 
-The notifications API is exposed via the BusNotifications class that can be injected via DI.
+The notifications API is exposed via the `BusNotifications` (NServiceBus Version 5) or `Notifications` (NServiceBus Version 6) class that can be injected via DI.
```

Comment:
Do we use the DI shorthand elsewhere in doco? It's probably better to be explicit to say "using the container" and link it to http://docs.particular.net/nservicebus/containers/